docs/technical_reference/unit_models/electrodialysis_1D.rst

```diff
@@ -120,7 +120,9 @@ are parameters that should be provided in order to fully solve the model.
    "Cell pair number", ":math:`n`", "cell_pair_num", "None", "dimensionless", 1
    "Current utilization coefficient", ":math:`\xi`", "current_utilization", "None", "dimensionless", 1
    "Channel height", ":math:`d`", "channel_height", "none", ":math:`m` ", 1
-   "Membrane areal resistance", ":math:`r`", "membrane_areal_resistance", "['cem', 'aem']", ":math:`\Omega m^2`", 2
+   "Membrane areal resistance independent on ion concentration", ":math:`r_{const}`", "membrane_areal_resistance_const", "['cem', 'aem']", ":math:`\Omega m^2`", 2
```

Comment:
```suggestion
   "Membrane areal resistance ion concentration-independent constant", ":math:`r_{const}`", "membrane_areal_resistance_const", "['cem', 'aem']", ":math:`\Omega m^2`", 2
```

Reply:
The original is more accurate because that is not a dimensionless constant but a physically meaningful areal resistance at infinitive electrolyte concentration.

Reply:
I was originally going to just suggest "Membrane areal resistance independent _of_ ion concentration" but then was matching the variable nomenclature which uses '_const' throughout which I assumed was in reference to constant parameter (not dimensionless constant).